activiti-services/activiti-services-identity-keycloak/src/main/java/org/activiti/services/identity/keycloak/SecurityConfig.java

```diff
@@ -74,6 +74,6 @@ protected void configure(HttpSecurity http) throws Exception {
         super.configure(http);
         http.authorizeRequests()
                 .antMatchers("/*").hasRole("user") //could optionally set these by autowiring and parsing keycloakSpringBootProperties
```

Comment:
Could do with looking at this a bit more but it's not actually something changing in this PR. In this file the PR just turns off basic auth.